File: Latihan A/C++/Ram.cpp

class Ram{
    private:
        int price, capacity;

    public:

        Ram(){

        }
        Ram(int price, int capacity){
            this->price = price;
            this->capacity = capacity;
        }

        //set semua variaber
        void setPricer(int price){
            this->price = price;
        }
        void setCapacityr(int capacity){
            this->capacity = capacity;
        }

        //get semua variabel
        int getPricer(){
            return this->price;
        }

        int getCapacityr(){
            return this->capacity;
        }
        //tampilan untuk ram
        void printRam(){
            cout <<"  Kapasitas Ram      : " << this->capacity << endl;
            cout <<"  Harga Ram          : " << this->price << endl;
        }
        ~Ram(){
            
        }
};

File: Latihan B/PYTHON/Ram.py
class Ram:
    
    __capacity = ""
    __price = 0

    # konstruktor
    def __init__(self, capacity="", price=0):
        self.__capacity = capacity
        self.__price = price

    # set untuk semua variable
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def setPrice(self, price):
        self.__price = price

    # get untuk semua variable
    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price
    
    # tampilan untuk ram
    def printRam(self):
        print("  Kapasitas Ram      : " + str(self.getCapacity()))
        print("  Harga Ram          : " + str(self.getPrice()))